security/pkg/nodeagent/caclient/providers/google/client.go

```diff
@@ -135,6 +135,10 @@ func (cl *googleCAClient) getTLSDialOption() (grpc.DialOption, error) {
 	return grpc.WithTransportCredentials(creds), nil
 }
 
+func (cl *googleCAClient) GetRootCertBundle() ([]string, error) {
```

Comment:
If we don't expect this method to be called on googleCAClient/CitadelClient, maybe we can add a log before return. That would make it easier for troubleshooting.

Reply:
We do expect it to be invoked. This is in a different PR https://github.com/istio/istio/pull/32649